frontend-gd/src/main/pages/log-in-page/fb-login.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
```

Comment:
All about FB and Google log in, but if we have plans of going live with this thing we should triple check all the rules/regulations for using FB/Google to make sure our asses are covered.